src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -131,7 +163,7 @@ public void itemStateChanged(ItemEvent e) {
         mainPanel.add(fuzzResultTableScrollPane);
 
         add(toolbar, BorderLayout.PAGE_START);
-        add(mainPanel, BorderLayout.CENTER);
+        add(mainPanel, BorderLayout.CENTER); 

```

Comment:
Basically all UI strings should be added to the relevant Messages.properties file, from there it will be translated to other languages by our participation in crowdin, you'd access or use the internationalized string using code like:  `Constant.messages.getString("report.write.overwrite.dialog.message");`